shared/bash_remediation_functions/include_lineinfile.sh

```diff
@@ -156,3 +156,10 @@ function auditd_config_set() {
 
     set_config_file "/etc/audit/auditd.conf" "$parameter" "$value" "true" "" "" "true" " = " "\s*=\s*"
 }
+
+function coredump_config_set() {
+    local parameter="$1"
+    local value="$2"
+
+    set_config_file "/etc/systemd/coredump.conf" "$parameter" "$value" "true" "" "" "true" "=" "\s*=\s*"
```

Comment:
Bash `lineinfile` functions also don't support `ini` configuration files. It is ok to leave like that.